packages/account-v2/src/containers/POICountrySelector/POICountrySelector.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { InlineMessage, Text } from '@deriv-com/ui';
+import CountrySelector from '../../components/CountrySelector/CountrySelector';
+
+export const POICountrySelector = () => {
+    return (
+        <div className='flex flex-col gap-10'>
+            <Text className='text-xs lg:text-sm' weight='bold'>
+                Your identity verification failed because:
```

Comment:
Lets add a TODO here to display this message based on the kycAuthSttaus API response.

You can check this IDV_ERROR_STATUS in packages/shared/src/utils/constants/poi-failure-codes.tsx

Reply:
Oriteee will add a TODO here.